recipes/playwright/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "playwright" %}
+{% set version = "1.11.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/playwright-python/archive/890a83306194e7b846a49c0f1d495913bb601cd4.tar.gz
+  sha256: 71eeb3dbc4fee5c457db98b350d7cab09a801284ea254d99c30fd496887476b8
+
+build:
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py<37]
+  binary_relocation: False
```

Comment:
can you clarify why this is needed? is playwright shipping some binaries (for browsers?)

Reply:
Playwright ships a driver for browsers to work